guides/micronaut-metrics-oci/src/main/resources/application.yml

```diff
@@ -0,0 +1,58 @@
+micronaut:
+  application:
+    name: micronautguide
+---
+#tag::http-services[]
+micronaut:
+  http:
+    services:
+      kucoin: # <1>
+        urls:
+          - "https://api.kucoin.com" # <2>
+#end::http-services[]
+---
+#tag::metrics[]
+micronaut:
+  metrics:
+    enabled: true # <1>
+    binders: # <2>
+      files:
+        enabled: true
+      jdbc:
+        enabled: true
+      jvm:
+        enabled: true
+      logback:
+        enabled: true
+      processor:
+        enabled: true
+      uptime:
+        enabled: true
+      web:
+        enabled: true
+    export:
+      oraclecloud:
+        enabled: true
+        namespace: change-me # <3>
```

Comment:
Why have `change-me` instead of `micronaut_guide`?

Reply:
I have changed this here: https://github.com/micronaut-projects/micronaut-guides/pull/1042/commits/b3155533a894a7bc3a3e31485b9c74aa8cab2604